whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java

```diff
@@ -872,6 +874,26 @@ public String getDescription(final ObjectType objectType) {
         }
     }
 
+    class PrefixlenSyntax implements AttributeSyntax {
+
+        private static final UrlValidator validator = new UrlValidator(new String[] {"https"});
+
+        @Override
+        public boolean matches(ObjectType objectType, String value) {
+            return validator.isValid(value);
+        }
+
+        @Override
+        public String getDescription(ObjectType objectType) {
+            return """
+                    Prefixlen is a self-published format for IP additional metadata.
```

Comment:
Remove first sentence "Prefixlen is a self-published format for IP additional metadata.".

Replace RFC8805 with draft-ietf-opsawg-prefix-lengths.

Reply:
Done